testing/sdk_utils.py

```diff
@@ -18,6 +18,7 @@
 
 log = logging.getLogger(__name__)
 
+TIMEOUT_SECONDS = 15 * 60
```

Comment:
This change could be reverted

Reply:
I am using it here https://github.com/mesosphere/dcos-commons/pull/2611/files#diff-6c805b3ab91ef126953b020997763e91R170 ; we can probably use it in someother classes as well in future, so just going to keep it `sdk_utils`.